app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -1614,16 +1619,70 @@ class ChatActivity :
     }
 
     private fun startPlayback(message: ChatMessage, doPlay: Boolean = true) {
-        if (!active) {
+        //if (!active) {
```

Comment:
Is there any reason why this commented out?
It's just a crutch for now and has to be improved in the future by providing controls to play voice messages while app is not in foreground. But having voice messages that "suddenly" start playing without to have controls would be disturbing for most users.

Reply:
I made this change so that you can play the first audio message of a series, then turn off the screen and listen to all its consecutives voice messages.
I tested it, it does not seem that voice messages get played randomly.

Reply:
By the way, I am adding a boolean `playInBackground` to startPlayback so that it ignores the active check only when the parameter is set to true.

Reply:
done in commit 50123bf